src/test/java/Pages/Hotels/LandingPage2.java

```diff
@@ -0,0 +1,254 @@
+    By signInPageBtnLocator = By.xpath("//button[text()='Sign in']");
+    By signInBtnLocator = By.xpath("//a[text()='Sign in']");
+    By signInErrorMessageLocator = By.xpath("//*[@class='uitk-error-summary-heading']");
+
+    //Signup page locators
+    By termsAndConditionsLocator = By.xpath("//*[text()='Terms and Conditions']");
+    By privacyStatementLocator = By.xpath("//*[text()='Privacy Statement']");
+    By pageHeaderTermsAndCondLocator = By.xpath("//*[text()='Terms and Conditions']");
+    By pageHeaderPrivacyLocator = By.xpath("//*[text()='Privacy Statement']");
+
+    // Feedbackpage
+
+    By submitBtnOnFeedback = By.xpath("//button[text()='Submit']");
+    By feedbackLinkLocator = By.xpath("(//div[@class='uitk-text uitk-type-300'])[12]");
+    By feedbackErrorMessageLocator = By.xpath("//p[text()='Please fill in the required information highlighted below.']");
+    By redDottedBoxLocator = By.id("required_box_page_rating");
+
+    // Other
+    By rewardNightPromoLocator = By.xpath("//span[contains(text(), 'every 10 nights')]");
+
+
+    // Sign in/out
+    public void clickSignInField() {findWebElementWithWait(topMenuSignInBtnLocator).click();;
+    }
+
+    public void clickSignInBtn() {findWebElementWithWait(signInBtnLocator).click();}
+
+    public void clickSignUpBtn() {findWebElementWithWait(signUpBtnLocator).click();}
+
+    public void clickSignInBtnOnSignInPage() {findWebElementWithWait(signInPageBtnLocator).click();}
+
+    //Sign up page
+    public void clickTermsAndCondOnSignUpPage() {findWebElementWithWait(termsAndConditionsLocator).click();}
+
+    public void clickPrivacyStatementSignUpPage() {findWebElementWithWait(privacyStatementLocator).click();}
+
+
+  //sign in page
+    public void enterInvalidEmail() {
+        type(enterEmailLocator,"dalenaiscool3943@gmail.com"); }
+
+    public void enterInvalidPassword() {
+        type(enterPasswordLocator,"techno85u8"); }
+
+    public boolean isSignInErrorMessageDisplayed() {
+        findWebElementWithWait(signInErrorMessageLocator);
+        return isElementDisplayed(signInErrorMessageLocator);
+    }
+
+    //feedback page
+   public void clickFeedbackPage() {
+       findWebElementWithWait(feedbackLinkLocator).click();
+       String landingPageHandle = getCurrentWindowHandle();
+       switchToSecondBrowserWindow(landingPageHandle);
+       ;
+         }
+    public void clickSubmitOnFeedback() {
+        findWebElementWithWait(submitBtnOnFeedback).click();
+    }
+
+    public boolean isFeedbackPageErrorMessageDisplayed() {return isElementDisplayed(feedbackErrorMessageLocator);}
+
+    public boolean isRedDottedBorderDisplayed() {
+        return isElementDisplayed(redDottedBoxLocator); }
+
+    // Check in/out  methods
+    public void clickCheckInBtn() {
+        clickIt(checkInBoxLocator);
+        Misc.pause(1);
+    }
+
+    public void clickCheckOutBtn() {
+        clickIt(checkOutBoxLocator);
+        Misc.pause(1);
```

Comment:
Instead of every hard coded wait, you can use explicit wait like waitUntilElementVisible or waitUntilElementClickable and pass the locator of the element that you waiting for.
Again if you are using FluentWait driver you don't need to use other waits all the time.